prow/jobs/jobs.go

```diff
@@ -63,26 +63,33 @@ func (ps Presubmit) RunsAgainstBranch(branch string) bool {
 	return false
 }
 
+// Postsubmit runs on a timer.
+type Postsubmit struct {
+	Name string        `json:"name"`
+	Spec *kube.PodSpec `json:"spec,omitempty"`
+}
+
 type JobAgent struct {
```

Comment:
They're not exact duplicates, since Postsubmits don't have to set any info on GitHub or have triggers/rerun commands. I think the way I did it leads to less code.